packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -241,24 +273,36 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
 
     let stickyTopHeight = 0;
     let stickyBottomHeight = 0;
-
-    stickyItems.forEach((sticky: Sticky) => {
-      const { isStickyTop, isStickyBottom } = sticky.state;
-      if (sticky.nonStickyContent) {
-        if (isStickyTop) {
-          stickyTopHeight += sticky.nonStickyContent.offsetHeight;
-        }
-        if (isStickyBottom) {
-          stickyBottomHeight += sticky.nonStickyContent.offsetHeight;
+    const placeholderUsedForStickyContentTop = this.usePlaceholderForSticky('top');
+    const placeholderUsedForStickyContentBottom = this.usePlaceholderForSticky('bottom');
+
+    if (placeholderUsedForStickyContentBottom || placeholderUsedForStickyContentTop) {
+      stickyItems.forEach((sticky: Sticky) => {
+        const { isStickyTop, isStickyBottom } = sticky.state;
+        if (sticky.nonStickyContent) {
+          if (
+            isStickyTop &&
+            placeholderUsedForStickyContentTop &&
+            !this.verifyStickyContainerBehavior('above', StickyContainerBehaviorType.StickyAlways)
```

Comment:
Should all these conditional checks on prop values occur only when the component's props are provided / modified in the relevant lifecycle hooks?

Reply:
it is needed when the scrolling happens and component becomes sticky/non-sticky